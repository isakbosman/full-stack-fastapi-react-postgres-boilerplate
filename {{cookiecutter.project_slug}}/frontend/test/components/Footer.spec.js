import React from 'react';

import Footer from 'components/Footer';

function setup() {
  return mount(<Footer />);
}

describe('Footer', () => {
  const wrapper = setup();

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
