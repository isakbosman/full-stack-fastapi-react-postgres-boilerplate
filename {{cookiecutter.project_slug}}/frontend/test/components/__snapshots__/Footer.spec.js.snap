// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render properly 1`] = `
.c1 {
  box-sizing: border-box;
  padding-top: 16px;
  padding-bottom: 16px;
  margin-left: auto;
  margin-right: auto;
  padding-left: 16px;
  padding-right: 16px;
  max-width: 1440px;
  position: relative;
  width: 100%;
}

.c2 {
  box-sizing: border-box;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0 {
  border-top: 0.1rem solid #ddd;
}

@media (min-width:768px) {
  .c1 {
    padding-left: 32px;
    padding-right: 32px;
  }
}

<FooterWrapper>
  <footer
    className="c0"
  >
    <Container
      py={3}
      verticalPadding={false}
    >
      <div
        className="c1"
      >
        <Flex
          alignItems="center"
          justifyContent="space-between"
        >
          <div
            className="c2"
          >
            <iframe
              frameBorder="0"
              height="20px"
              scrolling="0"
              src="https://ghbtns.com/github-btn.html?user=gilbarbara&repo=react-redux-saga-boilerplate&type=star&count=true"
              title="GitHub Stars"
              width="110px"
            />
            <iframe
              frameBorder="0"
              height="20px"
              scrolling="0"
              src="https://ghbtns.com/github-btn.html?user=gilbarbara&type=follow&count=true"
              title="GitHub Follow"
              width="130px"
            />
          </div>
        </Flex>
      </div>
    </Container>
  </footer>
</FooterWrapper>
`;
